README.md

```diff
@@ -1,7 +1,7 @@
 <!-- docs -->
 
 <p align="center">
-  <a href="//architect.io" target="blank"><img src="https://www.architect.io/img/logo.svg" width="480" alt="Architect Logo" /></a>
```

Comment:
Did this link stop working? If it's not working anymore that's a bit of a code smell with the way wordpress is handling redirects. Not your fault, but I'd want to address it with the vendor handling our wordpress site.

Reply:
[//architect.io/img/logo.svg]()
docs: no
github: no

[//docs.architect.io/img/logo.svg]()
docs: yes
github: no

[https://docs.architect.io/img/logo.svg]()
docs: yes
github: yes